VS_Project/Pokerapp/Pokerapp/Tableview.cshtml

```diff
@@ -10,7 +10,9 @@
     string table = urlCode[3];
     string roomCode = urlCode[2];
 
-    var namen = db.Query("SELECT Name FROM Players WHERE Room_Code = @0 AND Tbl_nr = @1", roomCode, table);
+    int table = Convert.ToInt32(Session["tablenumber"]);
```

Comment:
Je moet tablenumber niet in een session opslaan, als je dit doet krijg je altijd dezelfde table op andere pagina's die je open hebt staan. Hierdoor wordt het onmogelijk om te testen of alles werkt.
Bovendien heb je nu twee tablenumber / table variabelen waar de bovenste (die uit de URL wordt gehaald) de correcte manier is om dit te doen, aangezien dit aan de URL gebonden is.